realm-templates/fill-realm-templates.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+ROOT=`dirname $0`
+
+function buildRealmTemplate() {
+    jq -s '[.[]]' "$patient_realm" "$dotbase_realm" > "$realm_output_path"
```

Comment:
You are using global variables here instead of the function parameters. If you want to use the function parameters you will have to change this to:
```suggestion
    jq -s '[.[]]' "$1" "$2" > "$realm_output_path"
```
or
```suggestion
    local patient_realm=$1
    local dotbase_realm=$2
    jq -s '[.[]]' "$patient_realm" "$dotbase_realm" > "$realm_output_path"
```

Here is a stackoverflow answer which might clarify this: https://stackoverflow.com/questions/6212219/passing-parameters-to-a-bash-function